generators/python-v2/ast/src/__test__/__snapshots__/PythonFile.test.ts.snap

```diff
@@ -112,6 +112,17 @@ my_variable: ImportedClass.nested.attribute
 "
 `;
 
+exports[`PythonFile > Write duplicative import names 1`] = `
+"from .cars import Car
+from .vehicles import Car
```

Comment:
I have ideas for how to fix this – it seems the other language ast modules handle this case by prepending `_` to an alias name. 

So, this would become 
```
from .cars import Car
from .vehicles import Car as _Car
```

Is that right? If so, I can explore solutions that produce this end result